db.go

```diff
@@ -124,6 +124,8 @@ type DB struct {
 	blockCache *ristretto.Cache
 	indexCache *ristretto.Cache
 	allocPool  *z.AllocatorPool
+
+	testChan chan string
```

Comment:
we can choose a better name so that we could use it in future for other tests too

Reply:
I suspect that you mean something other than "for other tests", since otherwise, the name "testChan" would suffice. What is your intent here?

On the Zoom call, we discussed the name "syncChannel" (or syncChan to be a bit more succinct).